### PR TITLE
Feature/news

### PR DESCRIPTION
**Pull request by nikilark**
Added news and profile pages into project and they can accessed through home controller

### CODE REVIEW

**Comment by OlegAndrus12** [comment]
Actually, it`s ready to use. 
We shoud add services for this pages

**Review by OlegAndrus12 — APPROVED** [review]

File: WebApplication/Models/NewsViewModel.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+using System;
+using System.Collections.Generic;
+using WebApplication;
+namespace WebApplication.Models
```
Comment (arihilmir) [inline WebApplication/Models/NewsViewModel.cs:4]: Reformat to C# code standard

File: WebApplication/Models/Person.cs [hunk 2]
```diff
@@ -0,0 +1,20 @@
+using System;
+namespace WebApplication{
+    public class Person{
```
Comment (arihilmir) [inline WebApplication/Models/Person.cs:3]: Move this to a separate project with folder:

Core.DTO <- Core is a project, DTO - folder.

WebApp models are used for forms and validation.

File: WebApplication/Models/ProfileViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+using System;
+using System.Collections.Generic;
+using WebApplication;
+namespace WebApplication.Models
+{
+    public class ProfileViewModel : AppViewModel{
```
Comment (arihilmir) [inline WebApplication/Models/ProfileViewModel.cs:6]: ViewModel approach is not used in an ASP.NET application. 

Services approach is used: small classes that responsible for one thing. They are injected into controllers.
Create a separate project for this part.

File: Layouts/AdminPage/AdminDashboard.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
```
Comment (arihilmir) [inline Layouts/AdminPage/AdminDashboard.html:1]: Remove layouts from the project.

Btw, it is not in development. Why is it in your project?

File: WebApplication/Controllers/HomeController.cs [hunk 5]
```diff
@@ -16,14 +18,25 @@ public HomeController(ILogger<HomeController> logger)
...
         public IActionResult Index()
         {
-            return View();
+            DateTime time = DateTime.Now;
+            string photo = "news1.jpg";
+            string title = "📢Лекція «Блокчейн: як працює біткоїн»";
+            string text = "Спікер заходу: Роман Левкович, студент 3 курсу факультету прикладної математики та інформатики🔝\n🗓Теми лекції:\n👉 що таке блокчейн❓\n👉 як створюють біткоїн та як це працює❓\n⏰ 6 березня, з 16:30 – 18:00\n📍 ауд.270\n💫Реєстрація обов'язкова!👇";
+            News n = new News(0,title,photo,text,time);
+            List<News> l = new List<News>{n,n,n};
+            NewsViewModel nw = new NewsViewModel{User="Roman",NewsFeed=l};
```
Comment (arihilmir) [inline WebApplication/Controllers/HomeController.cs:27]: This is not a ViewModel

**Review by arihilmir — CHANGES_REQUESTED** [review]
Don't implement two different tasks in one branch. Split them into 2.

File: WebApplication/Models/ProfileViewModel.cs (hunk 3 above)
Comment (nikilark) [reply WebApplication/Models/ProfileViewModel.cs:6]: Its not ViewModel, it is the model of exact view

File: WebApplication/Models/Person.cs (hunk 2 above)
Comment (nikilark) [reply WebApplication/Models/Person.cs:3]: It was not the task

File: WebApplication/Controllers/HomeController.cs (hunk 5 above)
Comment (nikilark) [reply WebApplication/Controllers/HomeController.cs:27]: but model of view

File: Layouts/AdminPage/AdminDashboard.html (hunk 4 above)
Comment (nikilark) [reply Layouts/AdminPage/AdminDashboard.html:1]: what s wrong with doctype?

File: WebApplication/Models/Person.cs (hunk 2 above)
Comment (arihilmir) [reply WebApplication/Models/Person.cs:3]: Actually, it is. You were to implement this part. As I said, WebApplication/Models stores models that used in forms. DTOs should be in separate place.

File: WebApplication/Models/ProfileViewModel.cs (hunk 3 above)
Comment (arihilmir) [reply WebApplication/Models/ProfileViewModel.cs:6]: Then choose a proper name and take a look at a comment above.

File: Layouts/AdminPage/AdminDashboard.html (hunk 4 above)
Comment (arihilmir) [reply Layouts/AdminPage/AdminDashboard.html:1]: There's nothing wrong with doctype. It is wrong to have folder from a branch with ongoing development.

File: WebApplication/Controllers/HomeController.cs (hunk 5 above)
Comment (arihilmir) [reply WebApplication/Controllers/HomeController.cs:27]: Then, why `Person` is not called `PersonViewModel`?

**Pull request closed without merging**